ProcessMaker/processmaker
# feature/FOUR-8256

## Issue & Reproduction Steps
N/A

## Solution
N/A

## How to Test

- Go to admin / Auth Clients
- Select edit / create / delete and do the necessary steps
- Check the security_logs table for a new log "authClientUpdated" / "authClientCreated" / "authClientDeleted"

## Related Tickets & Packages
https://processmaker.atlassian.net/browse/FOUR-8256

## Code Review Checklist

ci:deploy
